data-model/publication-flow.md

```diff
@@ -15,7 +15,7 @@ Een publicatieaangelegenheid is gelinkt aan een dossier via `dossier:behandelt`.
 
 Bij het opstarten van een nieuwe publicatieaangelegenheid wordt de openingsdatum (`dossier:openingsdatum`) op de huidige datum/tijd ingesteld. De titel (`dct:title`) en korte titel (`dct:alternative`) worden overgenomen van de besluitsvormingsaangelegenheid indien die er is.
 
-Sommige eigenschappen van de publicatieaangelegenheid hebben betrekking op het doeldocument van de publication-flow (vb. type regelgevend document, wijze van publicatie). Momenteel zijn deze rechtstreeks op publicatieaangelegenheid gedefinieerd, maar mogelijk willen we hier in de toekomst toch een aparte resource voor definiëren.
+Sommige eigenschappen van de publicatieaangelegenheid hebben betrekking op het doeldocument van de publication-flow (vb. type regelgevend document, wijze van publicatie). Momenteel zijn deze rechtstreeks op publicatieaangelegenheid gedefinieerd, maar mogelijk willen we hier in de toekomst toch een aparte resource voor definiëren. ((TBC en hoort misschien eerder als eigenschap op dossierniveau))
```

Comment:
Regelgevend document is inderdaad een eigenschap die misschien meer op dossier thuishoort. Maar dat is iets dat dan eerder nog verder uitgekuisd moet worden aan de kant van de besluitvorming (zorgen dat besluitvormingsaangelegenheid en dossier uit elkaar getrokken worden, zorgen dat het duidelijk is welke van de stukken in het dossier het besluit zijn enz.). Tot dan zou ik het op de publicatieaangelegenheid houden zodat het (voorlopig) strikt gescheiden blijft van de besluitvormingsaangelegenheid.